env-var-docs/README.md

```diff
@@ -0,0 +1,69 @@
+# Environment Variable Documentation
+
+The main configuration file for the CiviForm server is in
+[application.conf](../server/conf/application.conf). The file is in [HOCON
+format](https://github.com/lightbend/config/blob/main/HOCON.md) which supports
+reading values at server start time from environment variables. This is done
+using the `${?SOME_ENV_VAR}` substitution syntax.
+
+To aid in deploying the CiviForm server, we ensure every environment variable
+referenced in the configuration file has appropriate documentation. This
+documentation lives in [env-var-docs.json](../server/conf/env-var-docs.json).
+See [./parser-package/README.md](./parser-package/README.md) for the expected
+structure of env-var-docs.json. As part of a CiviForm release, the
+documentation is rendered as markdown and added to our [documentation
+website](https://docs.civiform.us/it-manual/sre-playbook/server-environment-variables).
+
+## GitHub actions automations
+
+- [check_vars_documented.py](./check_vars_documented.py): Ensures all
+  environment variables referenced in application.conf are documented in
+  env-var-docs.json and that env-var-docs.json does not document any variables
+  not referenced in application.conf (ensures a 1:1 mapping). Runs on any PR
+  changing application.conf or env-var-docs.json.
+
+- [run_regex_tests.py](./run_regex_tests.py): Ensures that all regular
+  expression validation rules in env-var-docs.json compile and that all
+  provided regular expressions tests pass. Runs on any PR changing
+  application.conf or env-var-docs.json.
+
+- [generate_markdown.py](./generate_markdown.py): Generates markdown from
+  env-var-docs.json and submits it to our [documentation
+  repository](https://github.com/civiform/docs/tree/main/docs/it-manual/sre-playbook/server-environment-variables).
+  Runs as a part of our release automation.
+
+## Developer setup
+
+To contribute to any of the python files in this directory, ensure you have a
+system installation of python version 3.10 or greater:
+
+```sh
+$ python3 --version
+```
+
+We use [virtual python
+environments](https://docs.python.org/3/library/venv.html) to manage
+dependencies. To set up a virtual environment with the required dependencies,
+run `bin/env-var-docs-create-venv` from the repository root. To activate the
```

Comment:
Because the python stdlib module is named venv (https://docs.python.org/3/library/venv.html) I think it is more appropriate to reuse venv in names rather than expand it to virtual_env.

I have changed the README to "We use virtual python environments (venv) ..." to help make this association clearer.